src/components/Misc/DeleteOrResetModal.tsx

```diff
@@ -0,0 +1,130 @@
+import { Box, Button, Modal, Stack, Text, TextInput } from '@mantine/core';
+import classes from './styles/DeleteOrResetModal.module.css';
+import { ChangeEvent, useCallback, useState } from 'react';
+
+type BaseProps = {
+	isOpen: boolean;
```

Comment:
Can we give more descriptive names for the props. Like `isModalOpen` and `modalHeader`?

Reply:
Sure, I have updated some of the props to make them more descriptive.